starter-kits/product/_details.md

```diff
@@ -1,19 +1,24 @@
 % ## Background
+% - optional
 
-% ## What this product offers
-
-% ## Data description
+% ## Product purpose
+% - required
 
 % ## Applications
+% - optional
 
 % ## Technical information
+% - required
 
 % ## Lineage
+% - required
 
 % ## Processing steps
+% - optional
 
 % ## Software
+% - optional
 
-## References
+% ## References
```

Comment:
The references was commented because it is an optional field. All the optional fields are commented whereas all the required fields are not commented.

Reply:
I didn't realise that was the pattern. I like that suggestion. I have copied it.